### DISCUSSION TITLE
¿Dónde se localizan los juegos descargados? ¿En cuál carpeta están los juegos que descargo?

### DISCUSSION THREAD

**Discussion by Jermaine316** [bucanero/pkgi-ps3#49] [Q&A] [answered]
¡Hola! Llevo un buen rato buscando en mmCM el juego que descargue para pasarlo a un pendrive, pero no encuentro el pkg del juego en ninguna parte. Aún no lo instaló porque sé que el pkg se borraría, ¿alguien sabe dónde está la carpeta de descargas de juego de la tienda?

**Comment by bucanero** [comment, marked as answer]
```
/dev_hdd0/tmp/pkgi/
/dev_hdd0/vsh/task/
/dev_hdd0/vsh/game_pkg/
```
